src/nxdoc/getting-started/develop-with-nuxeo-platform.md

```diff
@@ -637,38 +642,86 @@ From a command line:
     }
     ```
 
-#### Send the Operation to Studio
+#### Send the Operation to Nuxeo Studio
 
 1. Build a JAR file (without running the tests); from the `contract-mgt-project` folder run:
 
-    ```bash
-    $ mvn -DskipTests package
-    ```
+  ```bash
+  $ mvn -DskipTests package
+  ```
 
-2. Deploy the JAR (`contract-mgt-project/contract-mgt-project-core/target/contract-mgt-project-core-1.0-SNAPSHOT.jar`) in your Nuxeo server by copying it to **$NuxeoServer/nxserver/bundles**, then restart your server.
+2. Link Your Local Project to Your Studio Project
 
-3. Go to the local automation documentation at `http://NUXEO_SERVER/nuxeo/site/automation/doc` (for example `http://localhost:8080/nuxeo/site/automation/doc`).
+  ```bash
+  $ nuxeo studio link
+    info You are going to link a Studio project to this project.
+  ? NOS Username: my-user
+  ? NOS Password: [hidden]
+  ? Studio Project: my-project
+  ? Do you want to update your Maven settings.xml file accordingly? Yes
+  ```
 
-4. In the Document category click **Contract Updater**, then click on the **JSON definition** link and copy the operation definition.
+  **Warning**: At this point, you just added your Nuxeo Connect password in `~/.m2/settings.xml` in clear text. This is not recommended, please read the following `Security Best Practise` section.
 
-5. In Nuxeo Studio go to **Settings** > **Registries** > **Automation Operations** and paste the operation definition into the `"operations": []` array, for example:
+3. Export Your Contributions to Nuxeo Studio
 
-    ```
-    { "operations": [
-     {
-      "id" : "Document.ContractUpdater",
-      "label" : "Contract Updater",
-      "category" : "Document",
-      "requires" : null,
-      "description" : "On a contract, sets the reminder date to three months after its start date.",
-      "url" : "Document.ContractUpdater",
-      "signature" : [ "document", "document", "documents", "documents" ],
-      "params" : [ ]
-     }
-    ] }
-    ```
+  ```bash
+  $ nuxeo studio export
+    info Building and exporting your contributions to 'my-project' Studio project.
+    info Contributions sucessfully exported to 'my-project' Studio project.
+  ```
+
+4. In Nuxeo Studio go to **Settings** > **Automatic Registries** > **Automation Operations**, and you should see the JSON definition of your Operation.
+
+  The operation is now available in Automation Chain editor, under the Document category.
+
+##### Note: Security Best Practise
+
+At this point, if you never setted up your password before, you just added your Nuxeo Connect password in `~/.m2/settings.xml` in clear text.
+
+It is strongly recommended that you encrypt your Studio password (aka your Nuxeo Connect account password):
```

Comment:
setted?
I would replace the two lines with a more generic sentence
>  It is strongly recommended that you encrypt the secrets stored in `~/.m2/settings.xml`.

And point to the official documentation: https://maven.apache.org/guides/mini/guide-encryption.html